File: Go/LR6.go
package main

import "fmt"

func main() {
	var choice int
	fmt.Scanln(&choice)

	switch choice {
	case 1:
		number11()
	case 2:
		number12()
	case 3:
		number13()
	case 4:
		AESOFB()
	case 5:
		gauss()
		solveSeidel()
	case 6:
		path()
	default:
		fmt.Println("Неизвестный выбор")
	}
}

File: C++/LR6.cpp
#include <iostream>
#include "1.h"
#include "2.h"
#include "3.h"
#include "4.h"
using namespace std;

int main() {
	setlocale(LC_ALL, "Rus");

	int k;
	cin >> k;

	switch (k)
	{
	case 1:
	{
		number11();
		number12();
		number13();
		break;
	}
	case 2:
	{
		AESOFB();
		break;
	}
	case 3:
	{
		gauss();
		solveSeidel();
		break;
	}
	case 4:
	{
		path();
		break;
	}
	default:
		break;
	}

}